build/scripts/contiv-vol-run.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+set -e
+
+# cleanup the older container
+docker rm -f "$1" &>/dev/null || :
+
+## test for shared mount capability
+if ! grep "MountFlags" /lib/systemd/system/docker.service | grep shared &>/dev/null
+then
+  echo 1>&2 "Docker cannot run volplugin in its current state."
+  echo 1>&2 "You must change docker's systemd MountFlags to equal 'shared'"
```

Comment:
thanks @erikh 

Actually, I was mistaken. Reading further even hyperkube requires `Mountflags` to be set to `shared`, so we seem to be good for now. Sorry for the distraction, but I ended up learning something here !